### PR TITLE
[ADD] business_requirement_sale_timesheet: Trace task with deliverables

### PR DESCRIPTION
**Pull request by martintorresvm**

### CODE REVIEW

**Comment by martintorresvm** [comment]
cc @pedrobaeza @Tardo

**Review by Tardo — APPROVED** [review]
LGTM, code review. I don't know why this is related with timesheet.

File: business_requirement_sale_timesheet/models/project_task.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+from odoo import fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    business_requirement_deliverable_id = fields.Many2one(
+        comodel_name='business.requirement.deliverable',
+        related='sale_line_id.business_requirement_deliverable_id',
```
Comment (martintorresvm) [inline business_requirement_sale_timesheet/models/project_task.py:12]: @Tardo FYI `sale_line_id` at `project.task` is the reason of dependency with `sale_timesheet`

File: business_requirement_sale_timesheet/README.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    :target: https://translation.odoo-community.org/projects/business-requirement-11-0/business-requirement-11-0-business_requirement_sale_timesheet
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/222/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module extends the functionality of Business Requirement Sale to allow
+you trace Task created from Sales Orders with origin Business Requirement Deliverables.
```
Comment (pedrobaeza) [inline business_requirement_sale_timesheet/README.rst]: s/origin/source or original

**Comment by martintorresvm** [comment]
Changes done @pedrobaeza @Tardo

File: business_requirement_sale_timesheet/models/project_task.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        readonly=True,
+        string='Deliverable',
+    )
+
+    @api.multi
+    def action_view_deliverable_id(self):
```
Comment (pedrobaeza) [inline business_requirement_sale_timesheet/models/project_task.py]: Remove `_id` suffix from the method name

File: business_requirement_sale_timesheet/views/project_task.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+            <xpath expr="//field[@name='date_last_stage_update']" position="after">
+                <field name="business_requirement_deliverable_id"
+                       groups="business_requirement.group_business_requirement_user"/>
+            </xpath>
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button
+                    type="object"
+                    name="action_view_deliverable_id"
+                    class="oe_stat_button"
+                    icon="fa-tag"
```
Comment (pedrobaeza) [inline business_requirement_sale_timesheet/views/project_task.xml]: On BRD, you are using `fa-tasks`. Why using another one here?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
I'm doing the changes for speeding up this.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**